src/navigation/stacks/main/index.tsx

```diff
@@ -1,34 +1,57 @@
 import React from 'react';
+import Icon from 'react-native-vector-icons/Entypo';
 
-import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { createMaterialBottomTabNavigator } from '@react-navigation/material-bottom-tabs';
 
+import CartScreen from 'features/cart/screen';
+import FavouritesScreen from 'features/favourites/screen';
 import HomeScreen from 'features/home/screen';
+import PurchasesScreen from 'features/purchases/screen';
 import SettingScreen from 'features/settings/screen';
 
-import { translate } from 'localization/hooks';
+import styles from './styles';
 
-export enum MainStackScreens {
-  'Home' = 'Home',
-  'Settings' = 'Settings',
-}
+const Tab = createMaterialBottomTabNavigator();
 
-export type MainStackParamList = {
-  [MainStackScreens.Home]: undefined;
-  [MainStackScreens.Settings]: undefined;
-};
-
-const Stack = createNativeStackNavigator<MainStackParamList>();
+const HomeIcon = ({ color }: { color: string }) => <Icon name="home" color={color} size={26} />;
+const PurchasesIcon = ({ color }: { color: string }) => (
+  <Icon name="bookmarks" color={color} size={26} />
+);
+const CartIcon = ({ color }: { color: string }) => (
+  <Icon name="shopping-cart" color={color} size={26} />
+);
+const FavoritesIcon = ({ color }: { color: string }) => (
+  <Icon name="heart" color={color} size={26} />
+);
+const SettingsIcon = ({ color }: { color: string }) => <Icon name="cog" color={color} size={26} />;
 
 const MainStack = () => {
   return (
-    <Stack.Navigator>
-      <Stack.Screen name={MainStackScreens.Home} component={HomeScreen} />
-      <Stack.Screen
-        name={MainStackScreens.Settings}
+    <Tab.Navigator
+      initialRouteName="Home"
+      barStyle={styles.bar}
+      labeled={false}
+      activeColor="#000"
```

Comment:
wdyt about creating a Colors palette so you can use them from that instance and keep them in sync when needed